src/js/app.js

```diff
@@ -9,7 +9,35 @@
    */
   ns.app = {
 
-    init : function () {
+    init: function () {
+      /**
+        * i18n which stands for internalization is being applied to Piskel to allow for all content in Piskel
+        * to be translated
+        *
```

Comment:
When acronyms are used, usually they are just in parenthesis next to the full word. Then you can freely use the acronym in the rest of the content.
```suggestion
        * Piskel supports Internalization(i18n) by providing translations of its UI elements.
        *
```